airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -119,71 +118,37 @@ def validate(self) -> StepResult:
         return self.success_result
 
 
-class VersionFollowsSemverCheck(VersionCheck):
```

Comment:
This is now checked in the `connectors_qa` package https://github.com/airbytehq/airbyte/pull/35322